specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -2205,6 +2662,98 @@
         }
       },
       "description": "List of private endpoint connections associated with SAN"
+    },
+    "Snapshot": {
+      "type": "object",
+      "required": [
```

Comment:
Is it okay to put them in swagger?

Reply:
If u mean required field, we need to have it to make sure that properties are displayed as required in PSH / CLI